tools/local-network-setup/README.md

```diff
@@ -1,78 +1,76 @@
-DKG local network setup tool
-========================
+# DKG local network setup tool
 
-This tool will help you set up a local DKG v6 network running with the Ganache blockchain. It is useful for development and testing purposes and is used internally by the OriginTrail core developers.
+This tool will help you set up a local DKG v6 network running with the Hardhat blockchain. It is useful for development and testing purposes and is used internally by the OriginTrail core developers.
 <br/>
 
-
 **Note: This tool is an internal tool used by the OriginTrail team and thus is developed for our workflow, meaning that it currently only supports MacOS**, but we encourage you to adapt it for your workflow as well.
 
+# Prerequisites
 
-Prerequisites
-=============
-
-* An installed and running triplestore (graph database)
-  * We recommend testing with GraphDB. In order to download GraphDB, please visit their official [website](https://graphdb.ontotext.com/). Alternatively other triple stores can be used (Blazegraph, Apache Jena and other RDF native graph databases)
-* An installed and running MySQL server
-* You should have installed npm and Node.js (v16) or higher
-
+-   An installed and running triplestore (graph database)
+    -   We recommend testing with Blazegraph. In order to download Blazegraph, please visit their official [website](https://blazegraph.com/). Alternatively other triple stores can be used (GraphBD, Apache Jena and other RDF native graph databases)
+-   An installed and running MySQL server
+-   You should have installed npm and Node.js (v16) or higher
```

Comment:
Node isn't tested for Node.js >v16

Reply:
When I set up first time it didn't work with v20 so I'll remove it